### ISSUE TITLE
add some active contributors

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4220]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#4220|
|Type|task|
|Created|2010-01-14 21:51:21Z|

I think [LICENSE](LICENSE) (or any other appropriate file) could use the following contributors:

 - jesstess
 - khorn
 - timallen
 - jonathanj

... (people i missed)

Attachments:

* [contributors.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/a72/a726ba86e89d42680f0ea45b4e458de6b4b209c6/9aeadb83e89615b149432341b06816668907ab9f.patch) (922 bytes) - added by thijs on 2010-01-15 19:29:33Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__4220 4220
type__task task
reporter__thijs thijs
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1263505881000000 1263505881000000
changetime__1263687142000000 1263687142000000
version__None None
owner__ 
cc__jonathanj cc__jesstess cc__khorn cc__TimAllen cc__thijs cc__exarkun cc__glyph
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun set owner to @thijstriemstra|
|-|-|

Sure, sounds good.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="washort's avatar" src="https://avatars.githubusercontent.com/u/947487?s=50" width="50" height="50">](https://github.com/washort)<a name="note_3"></a>|@washort set status to `closed`|
|-|-|

(In [28019]) Hooray, new developers. Fixes [#4220](https://github.com/twisted/twisted/issues/4220).

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
